programs/nodeos/main.cpp

```diff
@@ -60,9 +63,31 @@ void logging_conf_handler()
 
 void initialize_logging()
 {
+   auto success = fc::log_config::register_appender<dfuse::dm::log_appender>( "deep-mind" );
```

Comment:
@arhag Suggestions here about registering the appender, and the default config changeset below?

In the first version(s) of our patch, we had a forked `fc` sub-module, and we hijacked the default config to register the deep mind appender + logger.

We later move to a new library inside `eos` codebase directly, since it was much easier to work with (and to improve build performance). But at that point, we needed to also move registration of the appender + default logger config in the `nodeos` binary directly.

It never felt "good" to my and I'm wondering if there is something else that could be done here, or a better location where I could put that.

Reply:
I believe this should be registered via a `logging.json`. Add a new appender type to `fc` so that is possible. That new appender can be a new type of console appender (called say `dmlog`) or even a file appender without too much work.

Reply:
Moving the logger to `fc` will resolve all this, all this code would go away.